doc/source/ray-core/actors/task-orders.rst

```diff
@@ -0,0 +1,13 @@
+Actor Tasks Execution Orders
+============================
+
+Synchronous, single-threaded actors execute tasks according
+to their initial submission order. Any submitted task will not
+execute until previously submitted tasks have finished execution.
+Note that actor task execution could fail. Upon failure, the following
+tasks submitted to the same actor will not execute until the failed
+task is successfully retried or the retry is exhausted.
+
+:ref:`Async or threaded actors <async-actors>` does not guarantee the
+task execution order. For example, the system might execute a task
```

Comment:
Didn't you add ways to configure this? Or is it always unordered?

Reply:
https://sourcegraph.com/github.com/ray-project/ray/-/blob/python/ray/_raylet.pyx?L1607 today it's always unordered.